### PR TITLE
WIP: new navigation

### PR DESCRIPTION
**Pull request by goldsteinsveta**
[ ] place for root of the roots
[x] Bottom Tab navigation
[ ] Easier to reach and less annoying **Scan button**
[ ] Add Network Account should be more buttony
[ ] Place for future app/network **Settings** (i.g. Add Network, Update Metadata, Manage Network #406, #407, #302  etc)
[ ] better overview of the **derivations structure** (#488, #487 etc)
[ ] screen heading

### CODE REVIEW

**Comment by goldsteinsveta** [comment]
bottom bar screenshots

**Comment by Tbaut** [comment]
I like :)

**Comment by hanwencheng** [comment]
Bottom Tabs is good idea, but IMO scan is a button, but using it as a tab will be misleading, maybe changing the style somehow and make it outstand from others.

**Comment by goldsteinsveta** [comment]
> using it as a tab will be misleading
@hanwencheng FYI I intended to pack all related to signing screens in this tab: Scanner, Messages, Transactions etc

File: package.json [hunk 1]
```diff
@@ -53,16 +55,22 @@
...
     "react-native-randombytes": "^3.5.3",
+    "react-native-reanimated": "^1.4.0",
     "react-native-secure-storage": "git+https://github.com/paritytech/react-native-secure-storage.git",
     "react-native-svg": "^9.7.1",
+    "react-native-tab-view": "^2.11.0",
     "react-native-tabs": "^1.0.9",
     "react-native-vector-icons": "^6.6.0",
     "react-navigation": "^3.12.1",
+    "react-navigation-material-bottom-tabs": "^2.1.5",
+    "react-navigation-tabs": "^2.6.2",
```
Comment (hanwencheng) [inline package.json]: this one is not used anywhere

File: package.json [hunk 2]
```diff
@@ -53,16 +55,22 @@
...
     "react-native-keyboard-aware-scroll-view": "^0.9.1",
     "react-native-markdown-renderer": "^3.2.8",
+    "react-native-navbar-color": "^1.0.3",
+    "react-native-paper": "^3.3.0",
     "react-native-popup-menu": "^0.15.6",
     "react-native-randombytes": "^3.5.3",
+    "react-native-reanimated": "^1.4.0",
     "react-native-secure-storage": "git+https://github.com/paritytech/react-native-secure-storage.git",
     "react-native-svg": "^9.7.1",
+    "react-native-tab-view": "^2.11.0",
```
Comment (hanwencheng) [inline package.json]: react-native-tab-view is not used anywhere

**Pull request closed without merging**